include/bluetooth/services/ancs_c.h

```diff
@@ -0,0 +1,580 @@
+/*
+ * Copyright (c) 2012 - 2020 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
+ */
+
+#ifndef BT_GATT_ANCS_C_H_
```

Comment:
The infix `_GATT_` should be dropped from services.

The infix `_C_` should be dropped from services.
In cases where server and client would end up with the same name, we add `_client_`, but since we don't have a server and likely never will we can drop this completely.

Reply:
Done. BT_ANCS_CLIENT_H_